setup.py

```diff
@@ -34,68 +35,69 @@ def run_tests(self):
-      # the project's main homepage
-      url='https://github.com/robinandeer/chanjo-report',
-      packages=find_packages(exclude=('tests*', 'docs', 'examples')),
-      # if there are data files included in your packages
-      include_package_data=True,
-      package_data={
-          'chanjo_report': [
-              'server/blueprints/report/static/*.css',
-              'server/blueprints/report/static/vendor/*.css',
-              'server/blueprints/report/templates/report/*.html',
-              'server/blueprints/report/templates/report/layouts/*.html',
-              'server/blueprints/report/templates/report/components/*.html',
-              'server/translations/sv/LC_MESSAGES/*',
-          ]
-      },
-      zip_safe=False,
-      install_requires=[
-          'setuptools',
-          'chanjo>=4.1.0',
-          'Flask-WeasyPrint',
-          'cairocffi',
-          'lxml>=3.0',
-          'cffi',
-          'Flask',
-          'SQLAlchemy',
-          'Flask-Babel',
-          'tabulate',
-          'Flask-Alchy',
-          'Flask-SQLAlchemy==2.1',
-          'pymysql',
-      ],
-      tests_require=['pytest'],
-      cmdclass={'test': PyTest},
-      # to provide executable scripts, use entry points
-      entry_points={
-          'chanjo.subcommands.4': ['report = chanjo_report.cli:report'],
-      },
-      # see: http://pypi.python.org/pypi?%3Aaction=list_classifiers
-      classifiers=[
-          'Development Status :: 3 - Alpha',
-          'Intended Audience :: Developers',
-          'Topic :: Software Development',
-          'License :: OSI Approved :: MIT License',
-          'Programming Language :: Python :: 3.6',
-          'Environment :: Console'
-      ])
+setup(
+    name="chanjo-report",
+    # versions should comply with PEP440
+    version="4.7.0",
+    description="Automatically render coverage reports from Chanjo ouput",
+    long_description=LONG_DESCRIPTION,
+    # what does your project relate to?
+    keywords="chanjo-report development",
+    author="Robin Andeer",
+    author_email="robin.andeer@scilifelab.se",
+    license="MIT",
+    # the project's main homepage
+    url="https://github.com/robinandeer/chanjo-report",
+    packages=find_packages(exclude=("tests*", "docs", "examples")),
+    # if there are data files included in your packages
+    include_package_data=True,
+    package_data={
+        "chanjo_report": [
+            "server/blueprints/report/static/*.css",
+            "server/blueprints/report/static/vendor/*.css",
+            "server/blueprints/report/templates/report/*.html",
+            "server/blueprints/report/templates/report/layouts/*.html",
+            "server/blueprints/report/templates/report/components/*.html",
+            "server/translations/sv/LC_MESSAGES/*",
+        ]
+    },
+    zip_safe=False,
+    install_requires=[
+        "chanjo @ git+https://github.com/Clinical-Genomics/chanjo.git#egg=repo-4.6.1",  # Temporary fix due to outdated chanjo on PyPI
```

Comment:
This is the only line I've changed in this file. All the rest is the black reformatting and isort